### PR TITLE
[INLONG-4866][Manager] Support batch task for Inlong group

### LINKED ISSUE #4866: [Improve][Manager] Support batch task for Inlong group
### Description

_No response_

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kipshi**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #4866 

### Modifications

1 add batch.task flag for inlong group.
2 remove unused classes.

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/PushSortConfigListener.java [hunk 1]
```diff
@@ -1,109 +0,0 @@
...
-import org.springframework.stereotype.Component;
-
-import java.util.List;
-
-/**
- * Push sort config when enable the ZooKeeper
- */
-@Deprecated
-@Component
-public class PushSortConfigListener implements SortOperateListener {
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/PushSortConfigListener.java:45]: Please do not delete these old protocol classes, I need them to refactor some codes.

Comment (kipshi) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/PushSortConfigListener.java]: done

**Review by leosanqing — APPROVED** [review]

**Review by healchow — CHANGES_REQUESTED** [review]

**Comment by thesumery** [comment]
please add more description for motivation

**Review by thesumery — COMMENTED** [review]
please add more description for motivation

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/CreateSortConfigListener.java [hunk 2]
```diff
@@ -18,96 +18,195 @@
...
+                    createNodesForStream(sourceMap.get(streamId), sinkMap.get(streamId)),
+                    createNodeRelationsForStream(sourceMap.get(streamId), sinkMap.get(streamId)));
+            streamInfos.add(streamInfo);
         }
-        return ListenerResult.success();
+
+        return new GroupInfo(groupId, streamInfos);
+    }
+
+    private Map<String, List<StreamSource>> createPulsarSources(
```
Comment (haibo-duan) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/CreateSortConfigListener.java:143]: java doc

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/CreateSortConfigListenerV0.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            }
+            upsertDataFlow(groupInfo, extInfo);
+        } catch (Exception e) {
+            LOGGER.error("create sort config failed for sink list={} ", streamSinks, e);
+            throw new WorkflowListenerException("create sort config failed: " + e.getMessage());
+        }
+        return ListenerResult.success();
+    }
+
+    private void upsertDataFlow(InlongGroupInfo groupInfo, InlongGroupExtInfo extInfo) {
```
Comment (haibo-duan) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/CreateSortConfigListenerV0.java:108]: java doc

**Comment by woofyzhao** [comment]
Duplicated with https://github.com/apache/inlong/pull/5399

**Pull request closed without merging**